source/manage/cloud-billing.rst

```diff
@@ -4,10 +4,23 @@ Mattermost Cloud billing
 .. include:: ../_static/badges/allplans-cloud.rst
   :start-after: :nosearch:
 
-Mattermost Cloud is offered as a pay-as-you-go subscription service.
+From February 2023, Mattermost Cloud Professional and Enterprise plans are offered as an annual subscription service.
```

Comment:
@justinegeffen any thoughts on making this more specific, to say, February 2, 2023? I'm also open to keeping it as is

Reply:
Works for me!